lib/recurly.js

```diff
@@ -279,6 +279,65 @@ export class Recurly extends Emitter {
   }
 
   /**
+   * Pipelines a request across several requests
+   *
+   * Chunks are made of `data[by]`` in `size`. If any request errors,
+   * the entire pipeline errors
+   *
+   * - Ignores 404s until done
+   * - Immediately rejects other errors
+   * - Collects and concats array responses
+   * - Immediately resolves non-array responses
+   *
+   * @param {String} method
+   * @param {String} route
+   * @param {Object} data
+   * @param {Array} data.by must be an Array
+   * @param {String} by
+   * @param {Number} size=100
+   * @param {Function} done
+   * @private
+   */
+
+  pipedRequest ({ method, route, data, by, size }) {
```

Comment:
What about setting the default for `size` here like it appears in the doc block above. Since the only use of this function at this time using the default, we could remove the arg in that call. Is the `method` generally always going to be a `GET`? If so, we could set that default as well.

Reply:
I actually fixed the default assignment in #393 when I added specs for this method. I'll add it here too.
+1 to setting the method default as well